### PR TITLE
Mobile menu adjustments

### PR DESCRIPTION
**Pull request by rburkey87**
# Summary of Changes

**Fixes Issue #47**

• fixed padding within mobile menu items
• fixed padding/margins and color within mobile user-info
• fixed padding, display flex, and color within mobile action items
• fixed colors for mobile actions background/border

BEFORE:
image

AFTER:
image
image

# Browser Testing

**I have tested these changes in Chrome, Firefox, and Edge. Someone with a Mac should test it in Safari.

Note that Drupal users will see the wrong color of mobile action items text until we push the changes through for the theme (which overrides this css styling).

### CODE REVIEW

File: components/byu-menu/mixins.scss [hunk 1]
```diff
@@ -99,75 +97,229 @@
...
+//
+//  .byu-menu-more-menu {
+//    @include mobileMoreMenu();
+//    max-width: 100%;
+//  }
+//
+//  .byu-menu-more-items {
+//    position: static !important;
+//  }
+//}
```
Comment (kebowdog) [inline components/byu-menu/mixins.scss]: I'm not sure why we're adding in so many commented out lines here. Can we get rid of those?

File: components/byu-menu/style.scss [hunk 2]
```diff
@@ -72,4 +117,130 @@
...
+//}
+//
+//.byu-menu-more-visible .byu-menu-more-menu {
+//  @include moreMenuVisible();
+//}
+//
+//:host(.mobile-view) {
+//  @include mobileHost();
+//
+//}
```
Comment (kebowdog) [inline components/byu-menu/style.scss]: We should probably get rid of this commented out code as well.

File: docs/lib/marked-element/marked-element.html [hunk 3]
```diff
@@ -1,264 +0,0 @@
-<!--
```
Comment (kebowdog) [inline docs/lib/marked-element/marked-element.html]: It looks like the whole docs/lib folder is getting deleted here. I think @katrialesser ran into the same problem for some reason.

**Comment by rburkey87** [comment]
Yeah, we should be able to delete it. It’s the old code from when the mixins were still messed up and I wanted to make sure I didn’t break anything when I fixed it.

File: docs/lib/marked-element/marked-element.html (hunk 3 above)
Comment (rburkey87) [inline docs/lib/marked-element/marked-element.html]: not sure what to do about this one, but the comments are deleted now.

File: components/byu-menu/style.scss (hunk 2 above)
Comment (rburkey87) [inline components/byu-menu/style.scss]: done

File: components/byu-menu/mixins.scss (hunk 1 above)
Comment (rburkey87) [inline components/byu-menu/mixins.scss]: done

**Comment by ThatJoeMoore** [comment]
@rburkey87 Here's how to fix the deleted stuff in /docs/lib:

`bower install`
`gulp build`
`git add docs`

Then push the result.  That should take care of @kebowdog 's concerns.

**Comment by ThatJoeMoore** [comment]
@kebowdog @rburkey87 Can I request that we choose 'Squash and Merge' when we merge this?  To do so, click on the little dropdown icon next to the 'Merge' button once the change is approved.

**Pull request merged**